mentors/javascript/p.json

```diff
@@ -30,5 +30,14 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I'm a full-time software developer for ClickBus, mostly focused on frontend using frameworks like Vue and Symfony. On top of that I am quite familiar with Clojure, PHP and NodeJS when it comes to backend."
+  },
+  
+  {
+    "github_username": "ReckerPhildev",
```

Comment:
Your GitHub user name starts with `R`.  These changes are not in the correct file.  They should be in `R.json`.